pkg/console/operator/operator.go

```diff
@@ -175,10 +174,8 @@ func (c *consoleOperator) Sync(obj metav1.Object) error {
 
 	// all configs needed to do a sync
 	if err := c.handleSync(operatorConfig, consoleConfig, infrastructureConfig); err != nil {
```

Comment:
Status is handled fine within `handleSync()` & below. There is no need to handle here.